webapp/channels/src/sass/routes/_settings.scss

```diff
@@ -648,6 +648,12 @@
                     }
                 }
             }
+
+            // Remove this when MM-62778 is completed
```

Comment:
This is the other bonus fix. Currently, the navigation highlight is visible when you tab into the tab list, but it disappears as soon as you switch tabs. I thought that might have been caused by the previous PR, but it seems like that's a pre-existing issue